python/graphstorm/run/gsgnn_np/gsgnn_np.py

```diff
@@ -153,6 +153,9 @@ def main(config_args):
         model.prepare_input_encoder(train_data)
         embeddings = do_full_graph_inference(model, train_data, fanout=config.eval_fanout,
                                              task_tracker=tracker)
+        # Only save embeddings of nodes from target ntype(s).
+        # Embeddings of nodes from other ntype(s) are meaningless.
```

Comment:
Can we have more comments about "meaningless"? Because this will change the user interface of saved embeddings.

Reply:
added